response/response.go

```diff
@@ -1,22 +1,32 @@
 package response
 
+import (
+	"net/http"
+)
+
 type Response struct {
-	Headers map[string]string
+	headers map[string]string
 	Body    interface{}
 }
 
 type Headers map[string]string
 
 func New(body interface{}) *Response {
 	return &Response{
-		Headers: make(map[string]string),
+		headers: make(map[string]string),
 		Body:    body,
 	}
 }
 
 func (res *Response) WithHeaders(headers Headers) *Response {
 	for k, v := range headers {
-		res.Headers[k] = v
+		res.headers[k] = v
 	}
 	return res
 }
+
+func (res *Response) SetHeaders(w http.ResponseWriter) {
```

Comment:
I'm worried user will confuse WithHeaders & SetHeaders.

Reply:
I think the problem is `rocket` & `response` are dependent on each others

Reply:
We can:
- move response back into rocket.
- create Handle function for Response type

Reply:
What create Handler function for Response type mean?

Reply:
`Handle(w http.ResponseWriter)`

This solution has a problem is we are going to depend on package rocket. Then get a cycle.

Reply:
I think Handle is better than SetHeaders.
And I don't see any cycle import problem in here.

Reply:
We need to assert return type html, json these stuffs

Reply:
oh, I see.

Reply:
If we extract type to a subpackage, it will be better?

Reply:
Then it became hard to use, that's what I want to avoid.

Reply:
If we move it into `response`?

Reply:
oh, that will be great.